packages/yum/groonga-release.spec

```diff
@@ -41,7 +41,26 @@ Groonga release files
 %{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-groonga
 %{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-groonga-RSA4096
 
+%post
+if [ -e "/etc/system-release" ]; then
+  distribution=$(</etc/system-release)
+  if [ "`echo $distribution | grep 'Amazon Linux release 2'`" ]; then
+    cat <<EOR > %{_sysconfdir}/yum.repos.d/groonga.repo
+[groonga]
+name=Groonga for Amazon Linux 2 - \$basearch
+baseurl=https://packages.groonga.org/centos/7/\$basearch/
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-groonga
+       file:///etc/pki/rpm-gpg/RPM-GPG-KEY-groonga-RSA4096
+EOR
+  fi
+fi
```

Comment:
Can we move this to `%install` section and replace `%{__install} -Dp -m0644 groonga.repo %{buildroot}%{_sysconfdir}/yum.repos.d/groonga.repo`?